src/coreclr/md/compiler/assemblymd.cpp

```diff
@@ -444,7 +444,7 @@ STDMETHODIMP RegMeta::EnumExportedTypes(           // S_OK or error
     BEGIN_ENTRYPOINT_NOTHROW;
 
     HENUMInternal       **ppmdEnum = reinterpret_cast<HENUMInternal **> (phEnum);
-    HENUMInternal       *pEnum;
+    HENUMInternal       *pEnumTmp = 0;
```

Comment:
For all the changes in this PR:
* Can we please keep the pEnum name? It would make the change smaller.
* Can you please use NULL instead of 0 (at all places in this PR - initialization and clearing the variable)?